packages/kbn-es-query/src/kuery/node_types/node_builder.ts

```diff
@@ -10,23 +10,23 @@ import type { RangeFilterParams } from '../../filters';
 import { KueryNode, nodeTypes } from '../types';
 
 export const nodeBuilder = {
-  is: (fieldName: string, value: string | KueryNode) => {
+  is: (fieldName: string, value: string | KueryNode): KueryNode => {
     return nodeTypes.function.buildNodeWithArgumentNodes('is', [
       nodeTypes.literal.buildNode(fieldName),
       typeof value === 'string' ? nodeTypes.literal.buildNode(value) : value,
     ]);
   },
   or: (nodes: KueryNode[]): KueryNode => {
-    return nodes.length > 1 ? nodeTypes.function.buildNode('or', nodes) : nodes[0];
+    return nodes.length === 1 ? nodes[0] : nodeTypes.function.buildNode('or', nodes);
```

Comment:
Wouldn't this also work?
```suggestion
    return nodeTypes.function.buildNode('or', nodes);
```

Reply:
Spoke to you about this offline, but just adding a note here. Returning the first node is a slight optimization (don't need to build an extra node, and don't have to wrap the ES query in an additional unnecessary `bool` clause).